File: tests/testdata/codegen/output-name-conflict-pp/dotnet/output-name-conflict.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;

return await Deployment.RunAsync(() => 
{
    var config = new Config();
    var cidrBlock = config.Get("cidrBlock") ?? "Test config variable";
    return new Dictionary<string, object?>
    {
        ["cidrBlock"] = cidrBlock,
    };
});

File: pkg/codegen/testing/utils/testdata/output-name-conflict-pp/go/output-name-conflict.go
package main

import (
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		cfg := config.New(ctx, "")
		cidrBlock := "Test config variable"
		if param := cfg.Get("cidrBlock"); param != "" {
			cidrBlock = param
		}
		ctx.Export("cidrBlock", cidrBlock)
		return nil
	})
}
